src/components/loading/loading.js

```diff
@@ -112,6 +112,15 @@ class Loading extends BaseComponent {
     this.player.addEventListener(this.player.Event.ALL_ADS_COMPLETED, () => {
       this.props.updateLoadingSpinnerState(false);
     });
+
+    this.player.addEventListener(this.player.Event.FIRST_PLAY, () => {
+      this.setState({afterFirstPlay: true});
+      this.props.updateLoadingSpinnerState(false);
```

Comment:
Don't hide the spinner here since we didn't start toplay yet,
the PLAYER_STATE_CHANGED event handler will hide it on "playing"

Reply:
fixed